build-extr.sh

```diff
@@ -56,7 +56,7 @@ while getopts "j:f:" opt; do
 done
 
 # 构建命令
-CMD="./mvnw clean package -Dmaven.test.skip=true -P prod,${JDK_VERSION},flink-single-version,aliyun,${FLINK_VERSION},web"
+CMD="./mvnw clean package -Dmaven.test.skip=true -P prod,${JDK_VERSION},flink-single-version,scala-2.12,aliyun,${FLINK_VERSION},web"
```

Comment:
dev has deleted the scala version selection. The default is 2.12. There is no need to add it here.